de-lib/src/main/java/org/iplantc/de/diskResource/client/NavigationView.java

```diff
@@ -115,7 +113,15 @@
          */
         boolean isLoaded(Folder folder);
 
-        void refreshFolder(Folder folder);
+        boolean isPathUnderKnownRoot(String path);
+
+        /**
+         * Reloads the child folders under the given Folder for this view's TreeStore only.
+         * May trigger a re-selection of the currently selected folder.
```

Comment:
Even better :)
`May trigger a re-selection of the currently selected folder to initiate a re-load of the center panel, and possibly force initiation of lazy-loading.`